src/Interpreter.cpp

```diff
@@ -139,46 +139,6 @@ Interpreter::Interpreter(Module *M, TSOPSOTraceBuilder &TB,
   IL = new IntrinsicLowering(TD);
 }
 
-static sigjmp_buf cf_env;
-
-static void cf_handler(int signum){
-  siglongjmp(cf_env,1);
-}
-
-static void CheckedFree(void *ptr, std::function<void()> &on_error){
-#ifdef HAVE_MALLOPT
```

Comment:
Are these the only uses of `HAVE_MALLOPT`?
Also, check and remove `HAVE_MALLOC_H`  and `HAVE_VALGRIND_VALGRIND_H` above.